### PR TITLE
Add links to each proj in roadmap

### PR DESCRIPTION
**Pull request by StanimalTheMan**
I wonder if I should have links for overall sections like React, MVC, Angular and code reviews.

### CODE REVIEW

File: TCSA.V2/Helpers/ProjectsSubHelpers/SqlProjectsHelper.cs [hunk 1]
```diff
@@ -111,7 +111,7 @@ internal static List<Project> GetProjects()
                 ExperiencePoints = 10,
                 Description = "Start your SQL journey with easy exercises",
                 Area = Area.SQL,
-                Level = Level.Grey,
+                Level = Level.Brown,
```
Comment (TheCSharpAcademy) [inline TCSA.V2/Helpers/ProjectsSubHelpers/SqlProjectsHelper.cs:114]: Why are these changing? 🤔

File: TCSA.V2/Components/Dashboard/RoadmapCard.razor [hunk 2]
```diff
@@ -26,9 +26,30 @@
                 <div class="col-12">
                     @foreach (var task in Tasks)
                     {
-                        <img class="roadmap-image m-1 rounded-circle @(!task.IsCompleted ? "" : "task-completed border border-3 border-success opacity-50")"
-                             src='img/Cards/@task.IconUrl' width='50'
-                             title=@(!task.IsCompleted ? task.Name : $"{task.Name} - Completed") />
+                        if (task.Slug != null)
+                        {
+                            if (task.IsProject)
```
Comment (TheCSharpAcademy) [inline TCSA.V2/Components/Dashboard/RoadmapCard.razor:31]: Could this be simplified? It looks like both blocks are identical except for the href. You could use a ternary.

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@StanimalTheMan Good work! Just a couple of questions.

File: TCSA.V2/Helpers/ProjectsSubHelpers/SqlProjectsHelper.cs (hunk 1 above)
Comment (StanimalTheMan) [reply TCSA.V2/Helpers/ProjectsSubHelpers/SqlProjectsHelper.cs:114]: Good question.  I forgot to delete this as I was / still am wondering why the SQL challenges are not being linked properly.

File: TCSA.V2/Components/Dashboard/RoadmapCard.razor (hunk 2 above)
Comment (StanimalTheMan) [reply TCSA.V2/Components/Dashboard/RoadmapCard.razor:31]: Will look into the (Blazor) syntax some more.

**Pull request merged**